### PR TITLE
chore: upgrade to @netlify/functions v2

### PR DESCRIPTION
**Pull request by benmccann**
It's a major version of `@netlify/functions`, but I couldn't find any breaking changes in it - only a piece of added functionality, so I added a changeset here to make this a minor version of the adapter

### CODE REVIEW

File: .changeset/purple-boxes-wash.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/adapter-netlify': minor
+---
+
+chore: upgrade to @netlify/functions v2"
```
Comment (ghostdevv) [inline .changeset/purple-boxes-wash.md:5]:
```suggestion
chore: upgrade to @netlify/functions v2
```

Comment (ghostdevv) [reply .changeset/purple-boxes-wash.md:5]: Does this even need a changeset since it's a dev dependency of adapter-netlify??

**Review by ghostdevv — APPROVED** [review]
LGTM - I looked through https://github.com/netlify/functions/pull/408 and couldn't find any breaking changes either

File: .changeset/purple-boxes-wash.md (hunk 1 above)
Comment (benmccann) [reply .changeset/purple-boxes-wash.md:5]: Ah, no. We should drop the changeset. Good catch!

**Pull request merged**